conanfile.py

```diff
@@ -39,6 +39,12 @@ def config_options(self):
             del self.settings.compiler
             del self.settings.build_type
 
+    def configure(self):
+        """We only use the C compiler so ensure we don't need to rebuild if C++ settings change"""
+        if self.settings.os != "Windows":
```

Comment:
How come we need to keep the settings on Windows?

Reply:
On Windows we use a pre-compiled binary from python.org so we don't even use the compiler. You'll see a couple of lines above that the compiler and build type are deleted on Windows.